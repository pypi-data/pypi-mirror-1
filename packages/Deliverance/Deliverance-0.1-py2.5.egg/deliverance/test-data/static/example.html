<html>
 <head>
  <title>I am a title</title>
  <link rel="rss" href="/rss.xml" />
 </head>
 <body>Early text<p>Paragraph one</p>
   <p>Paragraph two</p>
   extra
 </body>
</html>
