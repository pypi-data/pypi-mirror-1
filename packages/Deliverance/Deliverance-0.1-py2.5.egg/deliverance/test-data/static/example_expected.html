<html>
 <head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <title>I am a title</title>
  <link rel="rss" href="/rss.xml" />
 </head>
 <body>
   Some text
   <div id="content">
   <p>Paragraph one</p>
   <p>Paragraph two</p>
   </div>
 </body>
</html>
